CONTRIBUTING.md

```diff
@@ -13,7 +13,8 @@ For those just getting started, consult this  [guide](https://help.github.com/ar
 - Use descriptive names (`namespace` is better than `ns`, `dynakube` is better than `dk`, etc.)
 - Avoid using `client.Client` for 'getting' resources, use `client.Reader` (also known as `apiReader`) instead.
   - `client.Client` uses a cache (or tries to) that requires more permissions than normally, and can also give you outdated results.
-
+- Do not create procedures or functions with more than two parameters except constructors and factory functions. Structs and interfaces exist for a reason.
```

Comment:
```suggestion
- Avoid creating procedures or functions with more than two parameters except constructors and factory functions. Structs, methods and interfaces exist for a reason.
```

In some cases its fine, like creating a struct for a single function (that is called once) is overkill, when you use the 'thing' multiple times then yeah writing a novel of parameters is not cool 😅

Reply:
changed to a compromise